.github/actions/contentful-environment/action.yml

```diff
@@ -0,0 +1,97 @@
+name: Contentful Create
+description: Creates a new Contentful environment if required
+
+inputs:
+  pr-number:
+    description: 'PR Number'
+    required: true
+  contentful-space-id:
+    description: 'Contentful Space ID'
+    required: true
+  contentful-master-env:
+    description: 'Contentful master environment'
+    required: true
+  contentful-token:
+    description: 'Contentful access token'
+    required: true
+  max-contentful-envs:
+    description: 'Max Contentful environments'
+    required: true
+
+outputs:
+  contentful-env:
+    description: 'Proposed Contentful environment'
+    value: ${{ steps.determine-environment.outputs.env-id }}
+  new-contentful-env-needed:
+    description: 'Whether we should/can create a new Contentful environment'
+    value: ${{ steps.new-contentful-env.outputs.should }}
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Install software
+      id: install-software
+      shell: bash
+      run: |
+        apt update
+        apt install -y jq
+        apt install -y curl
```

Comment:
We should already have curl on the Docker container.  But jq, if we need it we could install it on the image?

Reply:
https://github.com/yldio/asap-hub/blob/943583d6c7a416162bb30a8101aaf42b25711879/.github/Dockerfile#L10

Reply:
Good shout.